src/main/kotlin/com/safehill/kclient/network/remote/RemoteServer.kt

```diff
@@ -226,6 +227,22 @@ class RemoteServer(
 
     }
 
+    override suspend fun registerDevice(deviceId: String, token: String): String {
+        val bearerToken =
+            this.requestor.authToken ?: throw UnauthorizedSafehillHttpException
+
+        val userTokenRequest = UserDeviceTokenDTO(
+            deviceId = deviceId,
+            token = token,
+            tokenType = 1
```

Comment:
Should this use an enum?

Reply:
I tried making it an enum at first... but apns token type in android does not make sense because its never used in android. I can extract 1 to a const variable  as such:
```FCM_TOKEN_TYPE = 1```

Reply:
Sounds good